applications/vanilla/controllers/class.discussionscontroller.php

```diff
@@ -73,8 +74,9 @@ public function Table($Page = '0') {
     * @access public
     *
     * @param int $Page Multiplied by PerPage option to determine offset.
+    * @param array $where Discussions query filter.
     */
-   public function Index($Page = FALSE) {
+   public function Index($Page = FALSE, $where = array()) {
```

Comment:
We can't take a where clause in a controller method. A malicious user could pass something to this.